site/content/docs/user/quick-start.md

```diff
@@ -341,6 +341,20 @@ nodes:
 - role: worker
 ```
 
+#### Mounting files from host 
+You can mount files from the host system to the nodes with `extraMounts`:
+
+```yaml
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+# mount a path from the host to the control-plane node
+nodes:
+- role: control-plane
+  extraMounts:
+  - containerPath: /foo/path/on-the/node
+     hostPath: /path/on-the/host
+```
```

Comment:
I would add below `hostPath`this line `readOnly: true`. It can produce undesirable results if itś writable by default

Reply:
Added